UI/src/components/molecules/RipaLocation.vue

```diff
@@ -133,6 +143,38 @@
         </v-col>
       </v-row>
 
+      <template v-if="this.model.stopVersion === 2">
+        <ripa-subheader text="-- or --"></ripa-subheader>
+
+        <v-row no-gutters>
```

Comment:
Let's discuss if this should be at the top.  Also when I click current location, the "more location options" switch turns off and hides the field.